src/App.css

```diff
@@ -51,42 +51,34 @@
   font-size: 32px;
 }
 
-.MuiFab-root {
-  height: 18px !important;
+.MainGrid {
+  padding: 20px;
 }
 
-.MuiFab-primary {
-  border-radius: 0% !important;
-  width: auto !important;
-  height: 18px !important;
-  text-transform: none !important;
-  text-align: center;
-  box-shadow: none !important;
-  outline: none !important;
-}
-
-.MuiFab-label {
-  height: 18px !important;
+.ButtonPanel {
+  padding: 15px;
+  padding-top: 20px;
 }
 
-.MuiFab-primary p {
-  padding-left: 8px;
-  padding-right: 8px;
-  padding-top: 16px;
+.ButtonPanel-label {
+  display: inline-block;
+  position: relative;
+  top: 5px;
+  left: 10px;
+  opacity: 0.75;
 }
 
-.multiButtonOption {
-  transition: none !important;
-  margin: 0px !important;
-  box-shadow: none !important;
-
-  border-radius: 0% !important;
+.MuiPopover-paper {
+  z-index: 9999;
+  position: relative;
 }
 
-.multiButtonOption p {
-  padding-top: 16px;
+.MultiButton-main {
+  z-index: 9998;
```

Comment:
This seems silly to me. I understand using 999 to make sure something is in front but 1 and 2 should suffice for things that we don't need on top of absolutely everything.